src/backend/cuda/hooks/yaksuri_cudai_info_hooks.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "yaksi.h"
+#include "yaksuri_cudai.h"
+
+int yaksuri_cudai_info_create_hook(yaksi_info_s * info)
+{
+    int rc = YAKSA_SUCCESS;
+
+  fn_exit:
+    return rc;
+  fn_fail:
+    goto fn_exit;
+}
+
+int yaksuri_cudai_info_free_hook(yaksi_info_s * info)
+{
+    int rc = YAKSA_SUCCESS;
+
+  fn_exit:
+    return rc;
+  fn_fail:
+    goto fn_exit;
+}
+
+int yaksuri_cudai_info_keyval_append(yaksi_info_s * info, const char *key, const void *val,
+                                     unsigned int vallen)
+{
+    int rc = YAKSA_SUCCESS;
+
```

Comment:
It may be helpful to add a brief comment to note which state is updated when the `keyval_append` is implemented and maybe also on how it is update.

Reply:
I have a pending PR which will add one example.  That comment will be irrelevant at that point.

Reply:
That's perfect!